modules/distribution/src/repository/resources/dbscripts/apimgt/postgresql.sql

```diff
@@ -362,81 +364,81 @@ DROP TABLE IF EXISTS IDP;
 DROP SEQUENCE IF EXISTS IDP_SEQ;
 CREATE SEQUENCE IDP_SEQ;
 CREATE TABLE IDP (
-                  ID INTEGER DEFAULT NEXTVAL('IDP_SEQ'),
-                  TENANT_ID INTEGER,
-                  NAME VARCHAR(254) NOT NULL,
-                  IS_ENABLED CHAR(1) NOT NULL DEFAULT '1',
-                  IS_PRIMARY CHAR(1) NOT NULL DEFAULT '0',
-                  HOME_REALM_ID VARCHAR(254),
-                  IMAGE BYTEA,
-                  CERTIFICATE BYTEA,
-                  ALIAS VARCHAR(254),
-                  INBOUND_PROV_ENABLED CHAR (1) NOT NULL DEFAULT '0',
-                  INBOUND_PROV_USER_STORE_ID VARCHAR(254),
-                  USER_CLAIM_URI VARCHAR(254),
-                  ROLE_CLAIM_URI VARCHAR(254),
-                  DESCRIPTION VARCHAR (1024),
-                  DEFAULT_AUTHENTICATOR_NAME VARCHAR(254),
-                  DEFAULT_PRO_CONNECTOR_NAME VARCHAR(254),
-                  PROVISIONING_ROLE VARCHAR(128),
-                  IS_FEDERATION_HUB CHAR(1) NOT NULL DEFAULT '0',
-                  IS_LOCAL_CLAIM_DIALECT CHAR(1) NOT NULL DEFAULT '0',
-                      DISPLAY_NAME VARCHAR(255),
-                  PRIMARY KEY (ID),
-                  UNIQUE (TENANT_ID, NAME));
+			ID INTEGER DEFAULT NEXTVAL('IDP_SEQ'),
+			TENANT_ID INTEGER,
+			NAME VARCHAR(254) NOT NULL,
+			IS_ENABLED CHAR(1) NOT NULL DEFAULT '1',
+			IS_PRIMARY CHAR(1) NOT NULL DEFAULT '0',
+			HOME_REALM_ID VARCHAR(254),
+			IMAGE BYTEA,
+			CERTIFICATE BYTEA,
+			ALIAS VARCHAR(254),
+			INBOUND_PROV_ENABLED CHAR (1) NOT NULL DEFAULT '0',
+			INBOUND_PROV_USER_STORE_ID VARCHAR(254),
+ 			USER_CLAIM_URI VARCHAR(254),
+ 			ROLE_CLAIM_URI VARCHAR(254),
+  			DESCRIPTION VARCHAR (1024),
+ 			DEFAULT_AUTHENTICATOR_NAME VARCHAR(254),
+ 			DEFAULT_PRO_CONNECTOR_NAME VARCHAR(254),
+ 			PROVISIONING_ROLE VARCHAR(128),
+ 			IS_FEDERATION_HUB CHAR(1) NOT NULL DEFAULT '0',
+ 			IS_LOCAL_CLAIM_DIALECT CHAR(1) NOT NULL DEFAULT '0',
+	                DISPLAY_NAME VARCHAR(255),
+			PRIMARY KEY (ID),
+			UNIQUE (TENANT_ID, NAME));
 
 INSERT INTO IDP (TENANT_ID, NAME, HOME_REALM_ID) VALUES (-1234, 'LOCAL', 'localhost');
 
 DROP TABLE IF EXISTS IDP_ROLE;
 DROP SEQUENCE IF EXISTS IDP_ROLE_SEQ;
 CREATE SEQUENCE IDP_ROLE_SEQ;
 CREATE TABLE IDP_ROLE (
-                  ID INTEGER DEFAULT NEXTVAL('IDP_ROLE_SEQ'),
-                  IDP_ID INTEGER,
-                  TENANT_ID INTEGER,
-                  ROLE VARCHAR(254),
-                  PRIMARY KEY (ID),
-                  UNIQUE (IDP_ID, ROLE),
-                  FOREIGN KEY (IDP_ID) REFERENCES IDP(ID) ON DELETE CASCADE);
+			ID INTEGER DEFAULT NEXTVAL('IDP_ROLE_SEQ'),
+			IDP_ID INTEGER,
+			TENANT_ID INTEGER,
+			ROLE VARCHAR(254),
+			PRIMARY KEY (ID),
+			UNIQUE (IDP_ID, ROLE),
+			FOREIGN KEY (IDP_ID) REFERENCES IDP(ID) ON DELETE CASCADE);
```

Comment:
Directly copied content from https://github.com/wso2/carbon-apimgt/tree/v5.0.4/features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql as apim version upgraded to v5.04